package.json

```diff
@@ -22,7 +22,8 @@
     "react-dom": "^16.13.1"
   },
   "scripts": {
-    "generate-schema": "generate-airtable-schema && pretty-quick",
+    "add-require-to-airtable": "(echo \"require('dotenv-safe').config();\" && cat airtable/airtable.js) > temp.js && mv temp.js airtable/airtable.js",
```

Comment:
hmm is there a reason why we need to do this and can't just write these lines into the airtable.js file itself?

Reply:
because these lines only need to be added to the node version of this. If we added to airtable.js, they get overwritten by the schema generator everytime it is regenerated